README.md

```diff
@@ -90,192 +97,113 @@ let y = Named["string name"];
 
 ## Enum Declarations
 
-When an `enum` is declared, its members are evaluated and a new _enum constructor function_ is
+When an `enum` is declared, its members are evaluated and a new _enum object_ is
 created:
 
-  - When called, the constructor function coerces its argument to an enum value (similar to the 
-    behavior of `String`, `Number`, and `Boolean`), with precedence given to the earlier 
-    declarations: 
-    ```js
-    enum Numbers { zero, one, two, three, alsoThree = 3 }
-    Numbers(1) === Numbers.one
-    Numbers(3) === Numbers.three // (as opposed to `Numbers.alsoThree`)
-    ```
-  - When `new`-ed, the constructor function returns a wrapper `Object` that boxes the enum value
-    (Similar to the behavior of `String`, `Number`, and `Boolean`). 
-
-An _enum constructor function_ has a number of common methods, described in the [API](#API) section
-below.
-
-Enums can also be created programmatically via the global `Enum.create()` method, also described in
-the [API](#API) section.
+  - An _EnumDeclaration_ may have an `extends` clause whose value must be one of %Number%, 
```

Comment:
Possible extensibility point? Rather than specifically `%Number%`, `%String%`, or `%Symbol%`, it could be "anything which conforms to some protocol with a, for instance, `Symbol.enumMapping` where the `enumMapping` function is along the lines of `(x: number, key: string) => any`, and simply maps the numeric enum representation into whatever base is expected (which is called once on construction and whose results are cached into the enum instance slots). Eg, for `String`:
```js
String[Symbol.enumMapping] = (_, key) => key;
```
or `Symbol`:
```js
Symbol[Symbol.enumMapping] = (_, key) => Symbol(key);
```
or future `BigNum`:
```js
BigNum[Symbol.enumMapping] = i => new BigNum(i);
```

It would let you enum-up any class your heart desires (which could be useful for, eg, an enum `Direction` where each cardinal member is a singleton instance subclass of a root `Direction` base type with instance methods for comparing/working with them).

Just a thought.

Reply:
Might be a possible consideration, but I don't want to flood Stage 0/1 with too many new internal symbols. This already adds `@@formatEnum` and `@@parseEnum`.